File: src/sources/python/calculator.py
def calculator(a, b, operation):
    if operation == '+':
        return a + b
    elif operation == '-':
        return a - b
    elif operation == '*':
        return a * b
    elif operation == '/':
        return a / b if b != 0 else 'Деление на ноль'
    else:
        return 'Неизвестная операция'

print(calculator(10, 5, '+'))

File: src/sources/php/calculator.php
<?php

function calculator(float $a, float $b, string $operation): ?float
{
    return match ($operation) {
        '+' => $a + $b,
        '-' => $a - $b,
        '*' => $a * $b,
        '/' => $a / $b,
        default => null,
    };
}

echo calculator(10, 5, '+');